website/content/v1.10/kubernetes-guides/configuration/inlinemanifests.md

```diff
@@ -0,0 +1,176 @@
+
+    ```shell
+    cat << EOF > flux-extra-manifest.yaml
+    cluster:
+    extraManifests:
+        - "https://github.com/fluxcd/flux2/releases/latest/download/install.yaml"
+    EOF
+    ```
+
+1. Create a `CP_IPS` variable that contains the IP addresses of your control plane nodes:
+
+    ```bash
+    CP_IPS="<control-plane-ip-1>,<control-plane-ip-2>,<control-plane-ip-3>"
+    ```
+
+1. Run this command to export your `TALOSCONFIG` variable.
+You can skip this step if you've already done it:
+
+    ```bash
+    mkdir -p ~/.talos
+    cp ./talosconfig ~/.talos/config
+    ```
+
+1. Apply the `flux-extra-manifest.yaml` patch to your control plane nodes:
+
+    ```bash
+    talosctl patch machineconfig \
+    --patch @flux-extra-manifest.yaml \
+    --endpoints $CP_IPS \
+    --nodes $CP_IPS
+    ```
+
+1. Reboot the nodes.
+Note that if you have only one control plane node, rebooting it will cause cluster downtime.
+
+    ```bash
+    for NODE in $(echo "${CP_IPS}" | tr ',' ' '); do
+        echo "Rebooting control plane node: $NODE"
+        talosctl reboot --endpoints "$NODE" --nodes "$NODE" --wait
+    done
+    ```
+
+1. Wait a few seconds and check for the Flux pods:
+
+    ```bash
+    kubectl get pods -n flux-system -w
+    ```
+
+## Omni Patches
+
+You can also apply `inlineManifests` or `extraManifests` patches to Talos clusters managed by Omni.
+
+Refer to [Create a Patch For Cluster Machines](https://omni.siderolabs.com/how-to-guides/create-a-patch-for-cluster-machines?q=inline+manifest&ask=true) to learn how to create and apply the patches.
+
+## Summary: inlineManifests vs extraManifests
+
+Here’s a quick overview of the key differences between `inlineManifests` and `extraManifests`:
+
+|                        | `inlineManifests`                              | `extraManifests`                                             |
+| ---------------------- | ---------------------------------------------- | ------------------------------------------------------------ |
+| Source                 | Defined directly in the machine configuration  | Pulled from external URLs (GitHub gists, web servers, gists) |
+| Configuration Location | Under the `inlineManifests` section.           | Under the `extraManifests` section                           |
+| Usecase                | Early bootstrapping of critical resources      | For reusable, version-controlled, or shared manifests        |
+| Benefits               | No external dependencies                       | Centrally managed                                            |
+| Disadvantages          | Difficult to maintain and format embedded YAML | Requires external HTTP server                                |
+
+## How Talos Handles Manifest Resources During Upgrades
+
+During upgrades, Talos follows a conservative, additive-only approach when processing your `inlineManifests` and `extraManifests`.
```

Comment:
This should be rephrases, Talos reconciles manifests on every boot, on every failure to apply, on every change to the manifests in the machine config.

So "upgrade" there is irrelevant.